src/demo-app/button/button-demo.html

```diff
@@ -46,6 +46,20 @@
   </section>
 
   <section>
+    <button md-icon-button color="primary">
+      <i class="material-icons md-24">menu</i>
+    </button>
+
+    <button md-icon-button color="accent">
+      <i class="material-icons md-24">favorite</i>
+    </button>
+
+    <button md-icon-button>
+      <i class="material-icons md-24">more_vert</i>
+    </button>
```

Comment:
Can you add an icon-button to the disabled section at the bottom as well?  It will ensure disabled styles are applied properly to this type of button over time.